### PR TITLE
Adds Kimonos and Yukatas

### PR DESCRIPTION
**Pull request by Draggeru**
I'm suffering.

Adds Kimonos and Yukatas to the loadout menu, along with a black pair of sandals
additions
And of course, adds a fancy kimono set because I am an outfit whore
fancy

### CODE REVIEW

File: maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm [hunk 1]
```diff
@@ -217,3 +217,27 @@
 	icon_state = "penrosescarf"
 	inhand_icon_state = "penrosescarf"
 	resistance_flags = FIRE_PROOF|ACID_PROOF
+
+obj/item/clothing/head/costume/hairpin
```
Comment (CRITAWAKETS) [inline maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm:221]:
```suggestion
/obj/item/clothing/head/costume/hairpin
```

File: maplestation_modules/story_content/kimono/code/kimonoclothing.dm [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	icon = 'maplestation_modules/story_content/kimono/kimono_icon.dmi'
+	worn_icon = 'maplestation_modules/story_content/kimono/kimono_worn.dmi'
+	lefthand_file = 'maplestation_modules/story_content/kimono/kimono_lhand.dmi'
+	righthand_file = 'maplestation_modules/story_content/kimono/kimono_rhand.dmi'
+	digitigrade_file = 'maplestation_modules/story_content/kimono/kimono_digi.dmi'
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
+	body_parts_covered = CHEST|GROIN|ARMS
+	alternate_worn_layer = ABOVE_SHOES_LAYER
+	icon_state = "kimono1"
+	inhand_icon_state = "kimono1"
```
Comment (CRITAWAKETS) [inline maplestation_modules/story_content/kimono/code/kimonoclothing.dm:13]: kimono1 appears to be missing from kimono_lhand.dmi and kimono_rhand.dmi

Comment (Draggeru) [reply maplestation_modules/story_content/kimono/code/kimonoclothing.dm:13]: Ah fuck Cre never gave me the dmi for it, Ill update it when I can

Comment (Draggeru) [reply maplestation_modules/story_content/kimono/code/kimonoclothing.dm:13]: Nevermind I just reused the Yukata in hand sprites

**Pull request merged**